daemon/snap.go

```diff
@@ -154,10 +155,10 @@ func allLocalSnapInfos(st *state.State, all bool) ([]aboutSnap, error) {
 
 // appJSON contains the json for snap.AppInfo
 type appJSON struct {
-	Name         string   `json:"name"`
-	Daemon       string   `json:"daemon"`
-	DesktopFiles []string `json:"desktop-files"`
-	Aliases      []string `json:"aliases,omitempty"`
+	Name        string   `json:"name"`
+	Daemon      string   `json:"daemon"`
+	DesktopFile string   `json:"desktop-files,omitempty"`
```

Comment:
That said, do we need to expose this at all, given that we use a strong convention to find the file? In other words, it sounds trivial for Gnome Software to build the string internally instead of exposing this via an API that can't possibly be used without access to the underlying filesystem of snapd.